c2corg_ui/static/js/simplesearch.js

```diff
@@ -238,8 +237,12 @@ app.SimpleSearchController.prototype.createDataset_ = function(type) {
         return '<div class="header" dataset="' + type + '">' + this.gettextCatalog_.getString(type) + '</div>';
       }).bind(this),
       footer: function(doc) {
-        return '<p class="suggestion-more"><a href="/' + type + '/keyword/' + encodeURI(doc['query']) + '" class="green-text" translate>More results</a></p>';
-      },
+        if (!this.associationContext_) { // don't add this if you're typing in an add-association-tool
+          return '<p class="suggestion-more"><a href="/' + type + '?q=' + encodeURI(doc['query']) + '" class="green-text">+ see more results</a></p>';
```

Comment:
I have made changes at commit 	58ef63d
To make sure the i18n string is extracted I have also added it to ``c2corgui/templates/i18n.html``.